src/ruleLoader.ts

```diff
@@ -29,12 +29,12 @@ export interface IEnableDisablePosition {
 
 export function loadRules(ruleConfiguration: {[name: string]: any},
                           enableDisableRuleMap: {[rulename: string]: Lint.IEnableDisablePosition[]},
-                          rulesDirectory?: string): Lint.IRule[] {
+                          rulesDirectories?: string | Array<string>): Lint.IRule[] {
```

Comment:
this can be narrowed to just `string[]` once `ILinterOptions` is refactored (suggested below)